### ISSUE TITLE
Project Provisioning creates/provisions redundant Sonar Project with incorrect Master project key

### ISSUE THREAD

**Issue by IthaiS** [mibexsoftware/sonar-bitbucket-plugin#52]
### Expected Behavior

When enabling Project Provisioning, Quality profiles and/or Project Properties, it should just like always create 1 project in SonarQube with the correct key and branch identifier:

1. MyProjectName Feature_12345

### Actual Behavior

When enabling Project Provisioning, i.e. checking Quality profiles and/or Project Properties, it does exactly what it is supposed to do, but als creates a Sonar Project with the Master Project Key as additional project name

1. MyProjectName Feature_12345
2. MyProjectName develop Feature_12345 ==> this should not be happening

This results in semi-duplicate useless projects in SonarQube that need to be manually deleted.

### Plug-in version, SonarQube version, CI system, build type
Plug-in version: 1.11.3
SonarQube version: 6.2 - LGPL v3
CiSystem: Bamboo 5.15.3
Build type: happens on MSbuild and Maven

Regards,

Ithai.

**Comment by mrueegg** [comment]
Hi,

Thanks for your detailed bug report. I have a few questions because I'm not able to reproduce this:

1. Does this happen when you push new commits to an existing branch? If yes, to which one?

2. You mentioned that the plug-in creates a Sonar project with the Master project key as additional project name. But in the project under 2) you have `develop` in there. Could you please give me both the detailed project name and key of the wrongly created project?

3. Which Bitbucket Server version are you using?

Best regards,
Michael

**Comment by IthaiS** [comment]
Hi,

Thanks for getting back to us.
After spending more time carefully investigating this issue we noticed that we had a discrepency between the configuration of the BitBucket SonarQube plugin and the Bamboo SonarQube Task in regards to the "Escape invalid branch characters" check.
We now removed this for all projects and set it to the default (disabled, since we are now on SQ >5.0 anyway) on both BitBucket and Bamboo. This resulted in the errounous behavior going away. 
So basically it was a historical configuration that we now aligned everywhere.
I presume in time this setting will be removed.

Thank you for your help in this matter,

Ithai.
